### ISSUE TITLE
Can't enter custom invoice or Lightning Address when copied invoice on clipboard

### ISSUE THREAD

**Issue by moonshot75** [breez/breezmobile#738]
When there's an invoice on the clipboard you can't select another invoice or pay to a Lightning Address.
You have to clear your clipboard to continue with other payments after paying the one on the clipboard.

https://user-images.githubusercontent.com/66482965/151235817-b97fbc49-c659-4222-939b-e4472b18cad6.mp4

**Comment by kingonly** [comment]
The assumption is that if you want to pay another invoice or a lightning address, you will copy the info. It's rare that people actually type the destination. However, we'll see how we can improve it.

**Comment by moonshot75** [comment]
Might agree when you pay an invoice as you mostly copy that to your clipboard.
But if you have copied an invoice, paid it and then want to pay someone via Lightning Address it isn't possible.

**Comment by kingonly** [comment]
But how do get the lightning address? Do you type it manually? On mobile, you are likely to copy it and then you're fine...

**Comment by moonshot75** [comment]
Not sure, your data can tell after some time I guess.
I type it a lot as I know some out of my head, but maybe an exception.

**Comment by kingonly** [comment]
I'll keep this issue open to let others chime in and consider improving this, but from the data we have, currently this isn't an issue for our users.

**Comment by tlindi** [comment]
This is inconvinient. If one has expired invoice on clipboard, you cant go to invoice payment View.

Might it be possible to show reason of invalidy and present Clear Clipboard button on case of data on clipboard is not valid?

**Comment by kingonly** [comment]
My argument is that theoretically, that's right, but pragmatically it's hardly ever happens because people use copy/paste in mobile. Even if you can clear the clipboard, next you'll most like copy the new dest. We'll wait for further users input on this one.

**Comment by tlindi** [comment]
But it is real nasty to fix when such happens. Zap eg says nothing If clipboard has no compliant data And if there is valid invoice then it asks If you wanna use invoice.

**Comment by tlindi** [comment]
I'd like to see question if user would like to clear clipboard OR not to use data on clipboard.

See what happens cause Breez insists to use clipboard data. 

https://user-images.githubusercontent.com/7249767/221256374-5c6d93c8-7ebc-40a8-b07d-25bfda2f2d0a.mp4

User is not able to go and sent to Lightning Address cause clipboard stops him.

**Comment by kingonly** [comment]
I don't think we need a question. We should not prompt the dialog again. I need to rethink the paste UX.
